src/Passable.js

```diff
@@ -31,7 +31,7 @@ class Passable {
         return this.res;
     }
 
-    test = (fieldName: string, statement: string, ...args: [Severity, TestFn]) => {
+    test = (fieldName: string, statement: string, test: TestFn, severity: Severity) => {
```

Comment:
where the `Severity` type is defined? why I can't find it in the top of the file, with all the imports?